src/common/routing/RouteDefs.jsx

```diff
@@ -1,35 +1,71 @@
 import App from "App";
-import { Footer, Header, Home, PlayMeta, DefMeta, PageNotFound, PlayIdeas } from "common";
+import {
+  Footer,
+  Header,
+  Home,
+  PlayMeta,
+  DefMeta,
+  PageNotFound,
+  PlayIdeas,
+} from "common";
+import { connect } from "react-redux";
 import PlayList from "common/playlists/PlayList";
-import { getAllPlays } from "meta/play-meta-util";
 import { BrowserRouter, Route, Routes } from "react-router-dom";
+import { NhostApolloProvider } from "@nhost/react-apollo";
+import { NhostClient, NhostReactProvider } from "@nhost/react";
+import useGetPlays from "common/hooks/useGetPlays";
+import { useEffect } from "react";
+import { GET_ALL_PLAYS } from "../../actions/type";
 
-const RouteDefs = () => {
-  const plays = getAllPlays();
+const nhost = new NhostClient({
+  backendUrl: process.env.REACT_APP_NHOST_BACKEND_URL || "",
+});
+
+const ChildRoutes = (props) => {
+  const { dispatch, plays } = props;
+  const [_, error, data] = useGetPlays();
```

Comment:
@Angryman18 So this is the only place we load the plays data initially and then elsewhere from the store right?

Reply:
yes